src/main/java/org/complate/graal/GraalComplateRendererBuilder.java

```diff
@@ -0,0 +1,127 @@
+/**
+ * Copyright 2020 complate.org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.complate.graal;
+
+import org.complate.core.ComplateSource;
+import org.graalvm.polyglot.Context;
+import org.graalvm.polyglot.Value;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Function;
+import java.util.function.Supplier;
+
+import static org.graalvm.polyglot.HostAccess.ALL;
+
+/**
+ * Builder with sane default that has to be used to create an instance of
+ * {@link GraalComplateRenderer}.
+ */
+public final class GraalComplateRendererBuilder {
+
+    private static final Supplier<Context.Builder> DEFAULT_CONTEXT_BUILDER_FACTORY =
+        () -> Context.newBuilder("js")
+            .allowHostAccess(ALL)
+            .allowHostClassLookup((s) -> true)
+            .allowExperimentalOptions(true)
+            .option("js.experimental-foreign-object-prototype", "true");
```

Comment:
With the current code you would create a Graal based ComplateRenderer with `GraalComplateRenderer.of(source).withAdditionalCustomizations(builder -> builder.option(""polyglot.js.nashorn-compat", "true")).build();`

Reply:
Ok. I think exposing the Graal API here with the builder syntax is a good tradeoff, because we need to make tweaks like this (e.g setting "js.nashorn-compat") sometimes, but also most of the time will just want the sane defaults.

I also think the documentation is better after the last update.